swiftui/PasswordlessSwiftUI/PasswordlessSwiftUI/ContentView.swift

```diff
@@ -107,42 +112,43 @@ struct CustomStyledButton: View {
 
   var body: some View {
     Button(action: action) {
-      Spacer()
-      Text(title)
-        .padding()
-        .accentColor(.white)
-      Spacer()
+      HStack {
+        Spacer()
+        Text(title)
+          .padding()
+          .accentColor(.white)
+        Spacer()
+      }
```

Comment:
Nit: can you add a comment about why you're wrapping the text with two spacers in an H stack?